src/function.js

```diff
@@ -0,0 +1,70 @@
+export default function showAllTodo(tasks) {
+  const listElement = document.querySelector('.list');
+  listElement.innerHTML = '';
+
+  tasks.list.forEach((listItem) => {
+    listElement.innerHTML += `<div class="row" id="${listItem.index}" >
+            <div class="input">
+            <input type="checkbox" class="check" ${
+  listItem.completed ? 'checked' : ''
+}>
+            <input class="list-text" value="${listItem.description}" readonly>
+            </div>
+            <div class="icon">
+            <i class="fas fa-ellipsis-v"></i>
+            <i class="far fa-trash-alt hidden"></i>
+            </div>
+        </div>
+      `;
+  });
```

Comment:
rather than write HTML inside the .js, is better to create a template component (with a class that has `display: none` ) in the .HTML, then clone that template and modify the clone, this way you can benefit from linter and IDE helpers.